src/js/view/controls/components/timeslider.js

```diff
@@ -176,7 +176,7 @@ class TimeSlider extends Slider {
 
     onPlaylistItem(model, playlistItem) {
         this.reset();
-        model.set('cues', []);
+        this.updateCues(model, model.get('cues'));
```

Comment:
Since the cues are already empty, do we need this line anymore? `updateCues` will get called when cues are added.

Reply:
Yes, the `reset` clears all the cues, and the cues are being added from the ad plugins before this call

Reply:
@pajong - I think what Rob is saying is that whenever the cues on the model changes, `updateCues()` is called so we may not need this.